docs/_posts/2018-07-09-42.2.3-release.md

```diff
@@ -0,0 +1,156 @@
+* test: close of replication connection has not been fixed at backend side, so disable the test till 12.1 [PR 1243](https://github.com/pgjdbc/pgjdbc/pull/1243) [481460a3](https://github.com/pgjdbc/pgjdbc/commit/481460a32426c3d3a532f2a9a1b078ed9e98129a)
+* docs: prepare changelog for 42.3.3 release [99bbc510](https://github.com/pgjdbc/pgjdbc/commit/99bbc510f1be0176f10231a79da609941488d238)
+* chore: fetch contributors' URL via GitHub API [309101c3](https://github.com/pgjdbc/pgjdbc/commit/309101c39297868923102c477b16629ccc0dfed9)
+
+bazzargh (1):
+
+* fix: encode url query parameters DataSource [PR 1201](https://github.com/pgjdbc/pgjdbc/pull/1201) [9f3838f7](https://github.com/pgjdbc/pgjdbc/commit/9f3838f749d370a13a2fcef8e3ef67062d6e35eb)
+
+benbenw (2):
+
+* refactor: remove obsolete outParmBeforeFunc [PR 1234](https://github.com/pgjdbc/pgjdbc/pull/1234) [71028532](https://github.com/pgjdbc/pgjdbc/commit/71028532bcbc36e8239a4a7f9ad87e1acd070dc9)
+* perf: improve parsing performance of JDBC-style { call ...} calls [PR 1233](https://github.com/pgjdbc/pgjdbc/pull/1233) [435e2f79](https://github.com/pgjdbc/pgjdbc/commit/435e2f791bc848494b4f08c5d0b90ecf520ae5fe)
+
+benoit (3):
+
+* perf: reduce memory allocations when JDBC escapes ({fn ...}) are used [2a1e0910](https://github.com/pgjdbc/pgjdbc/commit/2a1e09100c4d56a37c84668135ec7fe3e05962cb)
+* refactor: use singleArgumentFunctionCall in EscapedFunctions [191d84eb](https://github.com/pgjdbc/pgjdbc/commit/191d84eb7541a0eba0a0f0eaac0f45e6e0c80ce4)
+* perf: avoid BaseQueryKey.toString in CachedQuery.getSize [PR 1227](https://github.com/pgjdbc/pgjdbc/pull/1227) [669fc31e](https://github.com/pgjdbc/pgjdbc/commit/669fc31ec187b27d15ee24f84c389260c54ddf25)
+
+bpd0018 (1):
+
+* style: rephrase comment on named portals [PR 1129](https://github.com/pgjdbc/pgjdbc/pull/1129) [86c46f94](https://github.com/pgjdbc/pgjdbc/commit/86c46f94535823ab7f6edde25f38d6e1182272a0)
+
+Étienne BERSAC (1):
+
+* docs: fix link to GitHub documentation [PR 1191](https://github.com/pgjdbc/pgjdbc/pull/1191) [655b6e70](https://github.com/pgjdbc/pgjdbc/commit/655b6e70b471da29b49124399eb0dab607dfc221)
+
+<a name="contributors_{{ page.version }}"></a>
+### Contributors to this release
+
+We thank the following people for their contributions to this release.
+
+[AlBundy33](https://github.com/AlBundy33)  
+[AlexElin](https://github.com/AlexElin)  
+[Dave Cramer](davec@postgresintl.com)  
+[Hari Babu Kommi](https://github.com/kommiharibabu)  
+[Jesper Pedersen](https://github.com/jesperpedersen)  
+[Jorge Solorzano](https://github.com/jorsol)  
+[KimBisgaardDmi](https://github.com/KimBisgaardDmi)  
+[Marc Slemko](https://github.com/znep)  
+[Michele Mancioppi](https://github.com/michele-mancioppi)  
+[Pavel Raiskup](https://github.com/praiskup)  
+[Sidi Mohamed EL AATIFI](https://github.com/elaatifi)  
+[Stephen Nelson](https://github.com/lordnelson)  
+[Vladimir Sitnikov](https://github.com/vlsi)  
+[bazzargh](https://github.com/bazzargh)  
```

Comment:
Looks like the sorting is case-sensitive. I'm not sure if case-insensitive would be better or not